script.js

```diff
@@ -85,4 +85,13 @@ function boxClick(e) {
             setTimeout(lightUpRandomSquare, 2000)
         }
     }
+}
+
+document.querySelector('.restartButton').addEventListener("click", restart => {
+    location.reload()
```

Comment:
page refresh? no! thats cheating and really slow. you need to run some code that restarts everything, try again